crates/xxi-node/src/message_handler.rs

```diff
@@ -658,10 +706,102 @@ fn read_tentenone_message<R: ::std::io::Read>(
         (SETTLE_CHANNEL_CONFIRM_TYPE, SettleConfirm),
         (SETTLE_CHANNEL_FINALIZE_TYPE, SettleFinalize),
         (RENEW_CHANNEL_OFFER_TYPE, RenewOffer),
+        (ROLLOVER_CHANNEL_OFFER_TYPE, RolloverOffer),
         (RENEW_CHANNEL_ACCEPT_TYPE, RenewAccept),
         (RENEW_CHANNEL_CONFIRM_TYPE, RenewConfirm),
         (RENEW_CHANNEL_FINALIZE_TYPE, RenewFinalize),
         (RENEW_CHANNEL_REVOKE_TYPE, RenewRevoke),
         (COLLABORATIVE_CLOSE_OFFER_TYPE, CollaborativeCloseOffer)
     )
 }
+
+#[cfg(test)]
+mod tests {
+    use crate::commons;
+    use crate::commons::ContractSymbol;
+    use crate::commons::Direction;
+    use crate::commons::FilledWith;
+    use crate::commons::OrderReason;
+    use crate::commons::OrderState;
+    use crate::commons::OrderType;
+    use crate::message_handler::TenTenOneMessage;
+    use crate::message_handler::TenTenOneMessageHandler;
+    use crate::message_handler::TenTenOneSettleOffer;
+    use crate::message_handler::WireMessage;
+    use crate::node::event::NodeEventHandler;
+    use dlc_manager::DlcChannelId;
+    use dlc_messages::channel::SettleOffer;
+    use lightning::ln::wire::CustomMessageReader;
+    use lightning::ln::wire::Type;
+    use lightning::util::ser::Readable;
+    use lightning::util::ser::Writeable;
+    use secp256k1::PublicKey;
+    use std::io::Cursor;
+    use std::str::FromStr;
+    use std::sync::Arc;
+    use time::OffsetDateTime;
+
+    #[test]
+    fn test_impl_serde_writeable() {
```

Comment:
🙃 Rename to `test_settle_offer_impl_serde_writeable`?

Thank you for adding a test, btw. We should do the same for all the messages with the help of `insta`. Then we can detect breaking changes here.

Reply:
I've added one additional test, but then stopped. Creating all this additional dummy data is just taking more effort than I'd like to invest right now.